resources/assets/coffee/react/beatmapset-page/details.coffee

```diff
@@ -0,0 +1,91 @@
+###
+# Copyright 2016 ppy Pty. Ltd.
+#
+# This file is part of osu!web. osu!web is distributed with the hope of
+# attracting more community
+# osu!web is free software: yocontributions to the core ecosystem of osu!.
+#
+# osu!web is free software: you can redistribute it and/or modify
+# it under the terms of the Affero GNU General Public License version 3
+# as published by the Free Software Foundation.
+#
+# osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+{div, p, span, hr, a, label, input} = React.DOM
+el = React.createElement
+
+class BeatmapsetPage.Details extends React.Component
+  DATE_FORMAT = 'MMM D YYYY'
+
+  constructor: (props) ->
+    super props
+
+    @state =
+      noVideo: false
+
+  render: ->
+    div className: 'page-contents__content beatmapset-details',
+      div className: 'beatmapset-details__avatar-container',
+        div
+          className: 'beatmapset-details__avatar avatar avatar--beatmapset'
+          style:
+            backgroundImage: "url(#{@props.set.author.data.avatarUrl})"
+
+      div className: 'beatmapset-details__mapper',
+        Lang.get 'beatmaps.beatmapset.show.details.made-by'
```

Comment:
username should be part of the translation. And the username should be a link. Check the one in `resources/assets/coffee/react/beatmap-discussions/system-post.coffee`.